src/main/java/echoServer/HTTP/HTTPResponse.java

```diff
@@ -0,0 +1,24 @@
+package echoServer.HTTP;
```

Comment:
I'll note that technically it's Java convention to use camelCase for package names (so this would be `http` not `HTTP`) and TitleCase for class names (so `HttpResponse` rather than `HTTPResponse`). But there's enough variability in how to handle acronyms, so I won't consider it a blocker. I would however encourage you to pay attention to naming conventions whenever you encounter a new language. 

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html